src/window.rs

```diff
@@ -18,19 +18,20 @@ use crate::gl;
 use glutin::dpi::{LogicalPosition, LogicalSize, PhysicalPosition, PhysicalSize};
 #[cfg(not(any(target_os = "macos", windows)))]
 use glutin::os::unix::EventsLoopExt;
-#[cfg(windows)]
+#[cfg(not(target_os = "macos"))]
 use glutin::Icon;
 use glutin::{
     self, ContextBuilder, ControlFlow, Event, EventsLoop, MouseCursor, PossiblyCurrent,
     WindowBuilder,
 };
-#[cfg(windows)]
+#[cfg(not(target_os = "macos"))]
 use image::ImageFormat;
 
 use crate::cli::Options;
 use crate::config::{Decorations, StartupMode, WindowConfig};
 
-#[cfg(windows)]
+// It's required to be in this direcroty due to `windows.rc` file
```

Comment:
```suggestion
// It's required to be in this directory due to the `windows.rc` file
```

I spotted another minor error.